css_tasks/task1/index.html

```diff
@@ -0,0 +1,273 @@
+                          <th><font face= "Montserrat">Price</font></th>
+                        </tr>
+                      </thead>
+                      <tbody>
+                        <tr>
+                          <td><font face ="Montserrat">1</font></td>
+                          <td><font face ="Montserrat">Sword</font></td>
+                          <td><font face ="Montserrat">10</font></td>
+                        </tr>
+                        <tr>
+                          <td><font face ="Montserrat">2</font></td>
+                          <td><font face ="Montserrat">Mace</font></td>
+                          <td><font face ="Montserrat">5</font></td>
+                        </tr>
+                        <tr>
+                          <td><font face ="Montserrat">3</font></td>
+                          <td><font face ="Montserrat">Faithful horse</font></td>
+                          <td><font face ="Montserrat">15</font></td>
+                        </tr>
+                      </tbody>
+                    </table>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+            <br>
+            <br>
+            <br>
+          </td>
+          <!-- Right Sidebar -->
+          <td width="198px" valign="top">
+            <table width="100%">
+              <tbody>
+                <!-- Inputs -->
+                <tr>
+                  <td>
+                    <table width="100%" cellpadding="5px">
+                      <tr>
+                        <td><font face="Montserrat" size="3">Login:</font></td>
+                      </tr>
+                      <tr>
+                        <td><input type="text" placeholder="Your login" /></td>
+                      </tr>
+                      <tr>
+                        <td>
+                          <font face="Montserrat" size="3">Password:</font>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td><input type="password" placeholder="*****" /></td>
+                      </tr>
+                    </table>
+                  </td>
+                </tr>
+                <!-- Dropdown -->
+                <tr>
+                  <td height="60px" valign="bottom">
+                    <table cellpadding="5px">
+                      <tr>
+                        <td>
+                          <select>
+                            <option value="admin">Admin</option>
+                            <option value="tester">Tester</option>
+                            <option value="user">User</option>
+                          </select>
+                        </td>
+                      </tr>
+                    </table>
+                  </td>
+                </tr>
+                <!-- Checkbox -->
+                <tr>
+                  <td>
+                    <table cellpadding="5px">
```

Comment:
More tables for god of tables =) Why so many?

Reply:
Yeah, it seems I was focused on including as many tables as I could :)